src/Features/Core/Diagnostics/DiagnosticData.cs

```diff
@@ -80,7 +81,7 @@ internal sealed class DiagnosticData
                     severity, severity, isEnabledByDefault, warningLevel,
                     ImmutableArray<string>.Empty, ImmutableDictionary<string, string>.Empty,
                     workspace, projectId, documentId, span,
-                    originalFilePath, originalStartLine, originalStartColumn, originalEndLine, originalEndColumn,
+                    null, originalStartLine, originalStartColumn, originalEndLine, originalEndColumn,
```

Comment:
That is the contract.


-          heejae

From: Srivatsn Narayanan [mailto:notifications@github.com]
Sent: Monday, April 6, 2015 5:52 PM
To: dotnet/roslyn
Cc: HeeJae Chang
Subject: Re: [roslyn] handle mapped file path properly (#1769)


In src/Features/Core/Diagnostics/DiagnosticData.cs<https://github.com/dotnet/roslyn/pull/1769#discussion_r27844372>:

> @@ -80,7 +81,7 @@ internal sealed class DiagnosticData

>                      severity, severity, isEnabledByDefault, warningLevel,

>                      ImmutableArray<string>.Empty, ImmutableDictionary<string, string>.Empty,

>                      workspace, projectId, documentId, span,

> -                    originalFilePath, originalStartLine, originalStartColumn, originalEndLine, originalEndColumn,

> +                    null, originalStartLine, originalStartColumn, originalEndLine, originalEndColumn,

I don't understand this - why are we passing null here and not originalFilePath?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/1769/files#r27844372>.